honeybee/model.py

```diff
@@ -1349,6 +1351,50 @@ def to_dict(self, included_prop=None, triangulate_sub_faces=False):
 
         return base
 
+    def to_hbjson(self, name="unnamed", folder_path=None, indent=0,
+                  included_prop=None, triangulate_sub_faces=False):
+        """Writes a Honeybee model to HBJSON.
+
+        Args:
+            name: A text string that will be the name of the HBJSON.
+                Defaults to "unnamed" for the file name for HBJSON.
+            folder_path: A text string of path to folder where HBJSON will be written.
+                Defaults to None. If folder_path is not specified, the default simulation
+                folder will be used to write the HBJSON. This default simulation folder
+                is at "C:\\Users\\USERNAME\\simulation."
```

Comment:
This is alright but, for future reference, this is only for Windows machines.  It's different on Mac and Linux.  So it's probably better not to state it in the docstring.

Reply:
Noted